ssntp/frame.go

```diff
@@ -291,3 +292,14 @@ func (f Frame) DumpTrace() (*payloads.FrameTrace, error) {
 
 	return &s, nil
 }
+
+// OriginUUID is the frame first sender and creator UUID.
+// When a SSNTP frame is forwarded by a server, the client
+// then only sees a new frame coming but it can not tell
+// who the frame creator and first sender is. This method
+// allows to fetch such information from a frame.
+func (f Frame) OriginUUID() (origin uuid.UUID) {
+	copy(origin[:], f.Origin[:16])
```

Comment:
I'm trying to figure out whether the [:16] slice is safe here.  As far as I can tell we're slicing a slice here and not an array, so there's no compile time guarantee that the slice will work.  But what I don't know is whether it's actually possible to for a SSNTP client to craft a frame with an Origin UUID of  less than 16 bytes?